cmd/perfguard/cmd_lint.go

```diff
@@ -1,10 +1,14 @@
 package main
 
 import (
+	"errors"
 	"flag"
+	"fmt"
 	"io"
 )
 
+var ErrIssuesFound = errors.New("found issues")
+
 func cmdLint(stdout, stderr io.Writer, args []string) error {
```

Comment:
Can we return the number of issues from cmdLint instead of encoding this knowledge into the error return value?
This way, we don't need a new global variable.

Reply:
yes, it's looks better